src/components/avatar/AvatarInitials.js

```diff
@@ -12,6 +12,7 @@ import {
     bdlWatermelonRed,
 } from '../../styles/variables';
 
+// NOTE: The same list exists in Avatar.scss
```

Comment:
This isn't ideal. It seems like all we need is the length of the list? Can we declare the list as a variable in `src/styles/constants/_colors.scss`, then import it here and in the Avatar.scss file?

Reply:
Maybe not there exactly - that should be a pretty pure file with just the palette definitions

Reply:
I was thinking of adding `src/styles/constants/_theme.scss` for aliases and derived values